openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/bigquery/create-credentials.md

```diff
@@ -3,6 +3,73 @@ title: BigQuery
 slug: /connectors/database/bigquery/create-credentials
 ---
 
+# Create custom role in GCP
```

Comment:
Could we add a couple of things here:

1. a small intro: What we are trying to do: 1) create a role 2) create a SA 3) associate both. IMO this small blurb should be part of the requirements in the index.md for the BQ page as well.
2. Instead of having titles that look as separate things, such as `Create custom role in GCP`, `Create Service Account and credential`, it might be helpful to specify the orders there in the titles:

```
<Intro>: we will blabla 1) create a role 2) create a SA 3) associate both
# 1. Create custom role in GCP
## Step 1 - blabla
# 2. Create Service Account and credential
## Step 1 - blabla
# 3. Bind SA and Role
```

wdyt

Reply:
will address these changes with separate pr